### PR TITLE
[stable/unifi] Docs: Fix name of cert secret

### PR DESCRIPTION
**Pull request by farmdawgnation**
### What this PR does / why we need it:

This PR fixes errant documentation in the unified chart.

#### Which issue this PR fixes

N/A

#### Special notes for your reviewer:

This is a simple documentation change so I didn't bother with bumping the chart version. I'm happy to do so if that's desired.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

@billimek @mcronce

### CODE REVIEW

**Comment by billimek** [comment]
Thanks @farmdawgnation, can you also please bump the chart version in `Charts.yaml` ?

**Comment by farmdawgnation** [comment]
Happy to! Just wasn't sure if I should for "just a doc change" :)

Doing it now.

**Review by billimek — APPROVED** [review]
LGTM

**Comment by farmdawgnation** [comment]
> ✖︎ unifi => (version: "0.10.0", path: "stable/unifi") > Chart version not ok. Needs a version bump!

Hm, I've missed something.

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.11.0
+version: 11.11.1
```
Comment (billimek) [inline stable/prometheus/Chart.yaml]: You bumped the `prometheus` chart and not the `unifi` chart, FYI!

Comment (farmdawgnation) [reply stable/prometheus/Chart.yaml]: Bahahahaha last week was rough

**Pull request merged**